script.module.resolveurl/lib/resolveurl/plugins/mixdrop.py

```diff
@@ -39,6 +39,10 @@ def get_media_url(self, host, media_id):
         if r:
             web_url = 'https://{0}{1}'.format(host, r.group(1))
             html = self.net.http_GET(web_url, headers=headers).content
+        else:
+            r = re.search(r'iframe.*src="([^"]+)', html)
+            web_url = 'https:{0}'.format(r.group(1))
+            html = self.net.http_GET(web_url, headers=headers).content
```

Comment:
Also this whole else section is unnecessary because `web_url = self.get_url(host, media_id)` already got this url and thats the page it opens

Reply:
This else statement is necessary because is what we need to translate that link with mixdrop.club into another link with mixdrop.co. The mixdrop.club link didn't follow the other scenarios.
When you open https://mixdrop.club/e/3ndm43mvtjxq4n/2/Piove_2022_DVDRip_AC3_ITA_CB01.mp4
the html contains this :
`<iframe width="100%" height="100%" src="//mixdrop.co/e/3ndm43mvtjxq4n" frameborder="0" scrolling="no" allowfullscreen ></iframe>`
and the regex `r = re.search(r'iframe.*src="([^"]+)', html)` 
get the new link "//mixdrop.co/e/3ndm43mvtjxq4n"
the `web_url = 'https:{0}'.format(r.group(1))` 
get the classic https://mixdrop.co/e/3ndm43mvtjxq4n and then we can proceed with `http_GET ` and 
`if '(p,a,c,k,e,d)' in html:`

Reply:
If you look at the code, ResolveURL will not open ` https://mixdrop.club/e/3ndm43mvtjxq4n/2/Piove_2022_DVDRip_AC3_ITA_CB01.mp4` it opens directly `https://mixdrop.co/e/3ndm43mvtjxq4n`

Reply:
> Okay redid it differently here [1056fb7](https://github.com/Gujal00/ResolveURL/commit/1056fb744ba0aea2b8ec0c379be4aea4a72ebbb3) Please test

I understand your edit and that's exactly what I thought before my pull request, but for me this is a simple workaround, and it is my opinion that there is no guarantee that all https://mixdrop.club links can be translated in https://mixdrop.co, this is not the way the browser takes when you open the link https://mixdrop.club. 
Are you 100% sure that another link with https://mixdrop.club but with a different videoid always points to https://mixdrop.co ?? 
If based on your experience the answer is YES then a simple replace may be fine.

Reply:
mixdrop.club seems to be a clone/fake of mixdrop.co. If they ever use anything other than mixdrop.co then it should be removed from this plugin. for now I will leave it in with the simple replace

Reply:
> mixdrop.club seems to be a clone/fake of mixdrop.co. If they ever use anything other than mixdrop.co then it should be removed from this plugin. for now I will leave it in with the simple replace

OK, I close this PR